packages/tests/src/commonlib/containeraAppValidator.ts

```diff
@@ -32,9 +34,20 @@ export class ContainerAppValidator {
     chai.assert.exists(this.subscriptionId);
     this.rg = getResourceGroupNameFromResourceId(resourceId);
     chai.assert.exists(this.rg);
-    this.containerAppName = this.ctx[EnvConstants.AZURE_CONTAINER_APP_NAME];
-    chai.assert.exists(this.containerAppName);
-    process.env[EnvConstants.AZURE_CONTAINER_APP_NAME] = this.containerAppName;
+    this.containerAppNames = [
+      EnvConstants.AZURE_CONTAINER_APP_NAME,
+      EnvConstants.BACKEND_APP_NAME,
+      EnvConstants.FRONTEND_APP_NAME,
+    ].filter((name) => this.ctx[name] && this.ctx[name].length > 0)
+    .map((name) => this.ctx[name]);
+
+    chai.assert.isTrue(
+      this.containerAppNames && this.containerAppNames.length > 0
+    );
+
+    process.env[ContainerAppValidator.CONTAINER_NAME_LIST] = JSON.stringify(
```

Comment:
Avoid Updating global environment variables, which may affect other test cases.

Reply:
Ok, I'll figure out how to optimize this.